docs/install/install-knative-gcp.md

```diff
@@ -55,11 +55,10 @@ export KO_DOCKER_REPO='gcr.io/my-gcloud-project-id'
    > if you are using Google Container Registry or `docker login` if you are
    > using Docker Hub.
 
-1. Install [Knative](https://knative.dev/docs/install/). Preferably, set up both
+1. Install [Knative](https://knative.dev/docs/install/). Set up
    [Serving](https://knative.dev/docs/serving/) and
-   [Eventing](https://knative.dev/docs/eventing/). The latter is only required
```

Comment:
Do we need eventing if we don't use Pub/Sub `Channel` or a `Broker` backed by a Pub/Sub
   `Channel`?

Reply:
According to https://github.com/google/knative-gcp/issues/1468#issuecomment-665286067 and https://github.com/google/knative-gcp/issues/1468#issuecomment-662111600 we need to install eventing but we might want to break this dependency in the future.

Reply:
`gcp-broker` uses Eventing's `broker` CRD, it also needs eventing to be installed.